packages/runner-ct/src/lib/state.ts

```diff
@@ -293,7 +293,7 @@ export default class State {
   }
 
   @action updateSpecByUrl (specUrl) {
-    const foundSpec = _.find(this.specs, { name: specUrl })
+    const foundSpec = _.find(this.specs, { name: decodeURI(specUrl) })
```

Comment:
I added this to be able to resolve the active spec from the URL when there are spaces

Reply:
Thanks

Reply:
good catch